src/main/kotlin/no/nav/syfo/dialogmote/domain/Dialogmote.kt

```diff
@@ -46,13 +46,15 @@ fun List<Dialogmote>.toArbeidstakerBrevDTOList(): List<ArbeidstakerBrevDTO> {
     return this.map { dialogmote ->
         val brevList = mutableListOf<ArbeidstakerBrevDTO>()
         dialogmote.referat?.let {
-            brevList.add(
-                it.toArbeidstakerBrevDTO(
-                    dialogmoteTidSted = dialogmote.tidStedList.latest()!!,
-                    deltakerUuid = dialogmote.arbeidstaker.uuid,
-                    virksomhetsnummer = dialogmote.arbeidsgiver.virksomhetsnummer,
+            if (it.ferdigstilt) {
```

Comment:
Viktig at mellomlagrede referater ikke tilgjengeliggjøres for arbeidstaker/arbeidsgiver.

Reply:
Hva med å kjøre en filter her først og så mappe. Altså først filtrere vekk dialogmote hvor ferdigstilt=false og å mappe. 
Resultatet blir jo det samme, men noe som kan gjøre det lettere å lese.

Reply:
Her skal det jo samles opp både referater og varsler (dvs innkallinger etc), så kan ikke filtrere bort møter der referat.ferdigstilt=false.

Så ser ikke helt hvordan dette kan endres slik du foreslår. Blir litt kunstig å filtrere noe anderledes for dialogmote.referat siden det er 0 eller 1.